EndlessClient/Dialogs/ScrollingListDialog.cs

```diff
@@ -74,7 +74,7 @@ public enum DialogType
         BankAccountDialog,
 
         Jukebox,
-        Barber,
```

Comment:
If `Barber` isn't going to be a `ScrollingListDialog` then this is fine, remove all references to `Barber` from this file